templates/bundles/PrestaShopBundle/Admin/Improve/Design/MailTheme/Blocks/generate_mails_form.html.twig

```diff
@@ -25,77 +25,74 @@
-              {{ form_widget(generateMailsForm.mailTheme) }}
-            </div>
+    <div class="card-block row">
+      <div class="card-text">
+
+        <div class="form-group row">
+          <label class="form-control-label">
+            {{ 'Select your email theme'|trans({}, 'Admin.Design.Feature') }}
+          </label>
+          <div class="col-sm">
+            {{ form_errors(generateMailsForm.mailTheme) }}
+            {{ form_widget(generateMailsForm.mailTheme) }}
           </div>
+        </div>
 
-          <div class="form-group row">
-            <label class="form-control-label">
-              {{ 'Select your language'|trans({}, 'Admin.International.Feature') }}
-            </label>
-            <div class="col-sm">
-              {{ form_errors(generateMailsForm.language) }}
-              {{ form_widget(generateMailsForm.language) }}
-            </div>
+        <div class="form-group row">
+          <label class="form-control-label">
+            {{ 'Select your language'|trans({}, 'Admin.International.Feature') }}
+          </label>
+          <div class="col-sm">
+            {{ form_errors(generateMailsForm.language) }}
+            {{ form_widget(generateMailsForm.language) }}
           </div>
+        </div>
 
-          <div class="form-group row">
-            <label class="form-control-label">
-              {{ ps.label_with_help(
+        <div class="form-group row">
+          {{ ps.label_with_help(
                 ('Select the theme you want to overwrite'|trans({}, 'Admin.Design.Feature')),
               ('PrestaShop\'s email templates are stored in the "mails" folder, but they can be overridden by your current theme\'s own "mails" folder. Using this option enables to overwrite emails from your current theme.'|trans({}, 'Admin.Design.Help'))
               ) }}
-            </label>
-            <div class="col-sm">
-              {{ form_errors(generateMailsForm.theme) }}
-              {{ form_widget(generateMailsForm.theme) }}
-              {% if generateMailsForm.theme.vars.disabled %}
+          <div class="col-sm">
+            {{ form_errors(generateMailsForm.theme) }}
+            {{ form_widget(generateMailsForm.theme) }}
+            {% if generateMailsForm.theme.vars.disabled %}
               <small class="form-text">{{ 'No emails were detected in any theme folder so this field is disabled.'|trans({}, 'Admin.Design.Help') }}</small>
-              {% endif %}
-            </div>
+            {% endif %}
           </div>
+        </div>
 
-          <div class="form-group row">
-            {{ ps.label_with_help(
+        <div class="form-group row">
+          {{ ps.label_with_help(
               ('Overwrite templates'|trans({}, 'Admin.Design.Feature')),
             ('By default, existing email template files are not modified to avoid deleting any modification you may have done. Enable this option to force the overwrite.'|trans({}, 'Admin.Design.Help'))
             ) }}
```

Comment:
```suggestion
            ('Overwrite templates'|trans({}, 'Admin.Design.Feature')),
            ('By default, existing email template files are not modified to avoid deleting any modification you may have done. Enable this option to force the overwrite.'|trans({}, 'Admin.Design.Help'))
          ) }}
```